README.md

```diff
@@ -1,66 +1,108 @@
-   and other WebAssembly execution strategies.
- - Develop a native ABI used for compiling WebAssembly suitable for use in both
-   JIT and AOT to native object files.
+If you've got the [Rust compiler
+installed](https://www.rust-lang.org/tools/install) then you can take some Rust
+source code:
 
-[Cranelift]: https://github.com/bytecodealliance/cranelift
-[Lightbeam]: https://github.com/bytecodealliance/wasmtime/tree/master/crates/lightbeam
+```rust
+fn main() {
+    println!("Hello, world!");
+}
+```
 
-#### Including Wasmtime in your project
+and compile/run it with:
 
-Wasmtime exposes an API for embedding as a library through the `wasmtime` subcrate,
-which contains both a [high-level and safe Rust API], as well as a C-compatible API
-compatible with the [proposed WebAssembly C API].
+```sh
+$ rustup target add wasm32-wasi
+$ rustc hello.rs --target wasm32-wasi
+$ wasmtime hello.wasm
+Hello, world!
+```
 
-For more information, see the [Rust API embedding chapter] of the Wasmtime documentation.
+## Features
 
-[high-level and safe Rust API]: https://docs.rs/wasmtime/
-[proposed WebAssembly C API]: https://github.com/WebAssembly/wasm-c-api
-[Rust API embedding chapter]: https://bytecodealliance.github.io/wasmtime/embed-rust.html
+* **Lightweight**. Wasmtime is a standalone runtime for WebAssembly that scales
+  with your needs. It fits on tiny chips as well as makes use of huge servers.
+  Wasmtime can be embedded into almost any application too.
 
-It's Wasmtime.
+* **Fast**. Wasmtime is built on the optimizing Cranelift code generator to
+  quickly generate high-quality machine code at runtime.
+
+* **Configurable**. Whether you need to precompile your wasm ahead of time,
+  generate code blazingly fast with Lightbeam, or interpret it at runtime,
+  Wasmtime has you covered for all your wasm-executing needs.
+
+* **WASI**. Wasmtime supports a rich set of APIs for interacting with the host
+  environment through the [WASI standard](https://wasi.dev).
+
+* **Standards Compliant**. Wasmtime passes the [official WebAssembly test
+  suite](https://github.com/WebAssembly/testsuite), implements the [official C
+  API of wasm](https://github.com/WebAssembly/wasm-c-api), and implements
+  [future proposals to WebAssembly](https://github.com/WebAssembly/proposals) as
+  well. Wasmtime developers are intimately engaged with the WebAssembly
+  standards process all along the way too.
+
+## Documentation
+
+[📚 Read the Wasmtime guide here! 📚][guide]
+
+The [wasmtime guide][guide] is the best starting point to learn about what
+Wasmtime can do for you or help answer your questions about Wasmtime.
+
+### License
+
+This project is licensed under the Apache License, Version 2.0,
+([LICENSE](LICENSE) or http://www.apache.org/licenses/LICENSE-2.0) with
+the [LLVM exception](https://spdx.org/licenses/LLVM-exception.html).
+
+### Contributing
+
+**[See the "Contributing" section of the guide for information on
+hacking on Wasmtime!][contributing]**
```

Comment:
This Contributing section is somewhat redundant with the Contributing link at the top, as they both link to the same place. Would it make sense to remove this section altogether, or perhaps add a sentence to the Documentation section mentioning that the guide has information about hacking on Wasmtime too?